torch/testing/_internal/common_methods_invocations.py

```diff
@@ -58,6 +58,16 @@ def __init__(self, input, *, args=tuple(), kwargs=None, output_process_fn_grad=N
         self.kwargs = kwargs if kwargs is not None else {}
         self.output_process_fn_grad = output_process_fn_grad
 
+    def __repr__(self):
+        arguments = [
+            f'input[{len(self.input)}]',
+            f'args={self.args}' if len(self.args) > 0 else None,
+            f'kwargs={self.kwargs}' if len(self.kwargs) > 0 else None,
+            (f'output_process_fn_grad={self.output_process_fn_grad}'
+             if self.output_process_fn_grad is not None else None)]
+
+        return f'SampleInput({", ".join(a for a in arguments if a is not None)})'
```

Comment:
This is slightly out of place, but it's useful for debugging `test_ops.py` test failures because you can see which sample is being run in the pytest traceback.

e.g. `SampleInput(input[1], args=(2,), kwargs={'rounding_mode': 'floor'})`